orchestrator-salt/src/main/resources/salt-common/salt/fluent/template/output_databus.conf.j2

```diff
@@ -15,14 +15,11 @@
     extra_headers_field              bundleContext
     event_message_field              message
     stream_name                      {{ fluent.dbusClusterLogsCollectionStreamName }}
-    partition_key                    "#{Socket.gethostname}"
-    {%- if fluent.proxyUrl %}
-    proxy_url                        "{{ fluent.proxyUrl }}"
-    {%- if fluent.proxyAuth %}
+    partition_key                    "#{Socket.gethostname}"{%- if fluent.proxyUrl %}
+    proxy_url                        "{{ fluent.proxyUrl }}"{%- if fluent.proxyAuth %}
     proxy_username                   "{{ fluent.proxyUser }}"
-    proxy_password                   "{{ fluent.proxyPassword }}"
-    {% endif %}
-    {% endif %}
+    proxy_password                   "{{ fluent.proxyPassword }}"{% endif %}{% endif %}{% if fluent.noProxyHosts and fluent.fluentVersion > 3 %}
+    no_proxy                         "{{ fluent.noProxyHosts }}"{% endif %}
```

Comment:
should be inside the {%- if fluent.proxyUrl %} but after {%- if fluent.proxyAuth %} 's endif